wagtailio/static/sass/components/_comparison-table.scss

```diff
@@ -0,0 +1,42 @@
+.comparison-table {
+    align-items: start;
+
+    td:first-child {
+        font-weight: $weight--bold;
+    }
+
+    tbody {
```

Comment:
suggestion: I think we should allow scrolling on the table only when the table is wider than the page - currently the whole page scrolls. It can be achieved with by adding the following:

```
    tbody {
        grid-column: 2 / span 2;
        grid-gap: 20px;
        overflow-x: scroll; // new

        @include media-query(medium) {
            grid-column: 2 / span 3;
            overflow-x: auto; // new
        }

```

Reply:
Neat!